src/object_store/cache.rs

```diff
@@ -576,7 +572,8 @@ mod tests {
 
         // Ensure that our TTL parameter is honored, so that we don't get a frozen cache after it
         // gets filled up once.
-        let _ = tokio::time::sleep(Duration::from_secs(1)).await;
+        let _ = tokio::time::sleep(Duration::from_millis(1100)).await;
```

Comment:
1.1s and not 1s? (funny I was complaining about `sleep` in tests in the previous PR and had it in my code to begin with)

Reply:
Oh, the previous `sleep` was put there by me (also 1s now leads to non-consistent results, sometimes a couple of entries get vacated and sometimes none at all)